src/toasts/UnverifiedSessionToast.ts

```diff
@@ -15,47 +15,50 @@ limitations under the License.
 */
 
 import { _t } from '../languageHandler';
+import dis from "../dispatcher/dispatcher";
 import { MatrixClientPeg } from '../MatrixClientPeg';
 import Modal from '../Modal';
 import DeviceListener from '../DeviceListener';
 import NewSessionReviewDialog from '../components/views/dialogs/NewSessionReviewDialog';
 import ToastStore from "../stores/ToastStore";
 import GenericToast from "../components/views/toasts/GenericToast";
+import { Action } from "../dispatcher/actions";
+import { USER_SECURITY_TAB } from "../components/views/dialogs/UserSettingsDialog";
 
 function toastKey(deviceId: string) {
     return "unverified_session_" + deviceId;
 }
 
-export const showToast = (deviceId: string) => {
+export const showToast = async (deviceId: string) => {
     const cli = MatrixClientPeg.get();
 
     const onAccept = () => {
-        Modal.createTrackedDialog('New Session Review', 'Starting dialog', NewSessionReviewDialog, {
-            userId: cli.getUserId(),
-            device: cli.getStoredDevice(cli.getUserId(), deviceId),
-            onFinished: (r) => {
-                if (!r) {
-                    /* This'll come back false if the user clicks "this wasn't me" and saw a warning dialog */
-                    DeviceListener.sharedInstance().dismissUnverifiedSessions([deviceId]);
-                }
-            },
-        }, null, /* priority = */ false, /* static = */ true);
+        DeviceListener.sharedInstance().dismissUnverifiedSessions([deviceId]);
+        dis.dispatch({
+            action: Action.ViewUserSettings,
+            initialTabId: USER_SECURITY_TAB,
```

Comment:
Hmm, have you added a way to start verification from settings...?

Reply:
nope, there was one already.

Reply:
Wait, where...? It's not in the session list AFAICT... Anyway, the new flow is still okay without it, I just wanted to be sure of your intention here.

Reply:
there's a link to your memberinfo from the top of the device section.

Reply:
Ah right, it's a bit indirect, but anyway it's at least possible to jump where needed.